app/scripts/models/verification/complete-2fa.js

```diff
@@ -0,0 +1,28 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/**
+ * A model to hold report-sign-in data.
+ */
+
+define(function (require, exports, module) {
+  'use strict';
+
+  const Vat = require('lib/vat');
+  const VerificationInfo = require('./base');
+
+  module.exports = VerificationInfo.extend({
+    defaults: {
+      code: null,
+      type: null,
```

Comment:
is `type` set in the query params that are sent in the push notification?

Reply:
Not yet. For now, only code and uid are sent.